plugin/othttp/handler.go

```diff
@@ -130,7 +130,7 @@ func NewHandler(handler http.Handler, operation string, opts ...Option) http.Han
 	h := Handler{handler: handler, operation: operation}
 	defaultOpts := []Option{
 		WithTracer(global.TraceProvider().Tracer("go.opentelemetry.io/plugin/othttp")),
-		WithPropagator(trace.TraceContext{}),
+		WithPropagator(trace.DefaultPropagator),
```

Comment:
There is another mention of TraceContext in this file, in a comment. Could you update it to `DefaultPropagator` too?

Reply:
done